<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoWidget</class>
 <widget class="QWidget" name="UserInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>584</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>764</width>
    <height>584</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>764</width>
    <height>584</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>在线网络教学系统客户端</string>
  </property>
  <property name="windowIcon">
   <iconset resource="educationsystem.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonMin">
       <property name="styleSheet">
        <string notr="true">QPushButton#buttonClose
{
    background-color:transparent;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="styleSheet">
        <string notr="true">QPushButton#buttonClose
{
    background-color:transparent;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;幼圆&quot;;</string>
       </property>
       <property name="text">
        <string>在线网络教学系统</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>171</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;幼圆&quot;;</string>
         </property>
         <property name="title">
          <string>基本信息</string>
         </property>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>111</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>当前用户：</string>
          </property>
         </widget>
         <widget class="QLabel" name="usernameLabel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>130</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="currentTimeLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>141</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>text</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>171</width>
           <height>311</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;幼圆&quot;;</string>
         </property>
         <property name="title">
          <string>用户管理</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QPushButton" name="learningTestButton">
            <property name="minimumSize">
             <size>
              <width>111</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:5px;
}
</string>
            </property>
            <property name="text">
             <string>学习特征调查问卷</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modifyButton">
            <property name="minimumSize">
             <size>
              <width>111</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:5px;
}
</string>
            </property>
            <property name="text">
             <string>修改用户信息</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="minimumSize">
             <size>
              <width>111</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:5px;
}
</string>
            </property>
            <property name="text">
             <string>保存用户信息</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="quitButton">
            <property name="minimumSize">
             <size>
              <width>111</width>
              <height>31</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>11</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:5px;
}
</string>
            </property>
            <property name="text">
             <string>退出用户管理</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>171</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;幼圆&quot;;</string>
         </property>
         <property name="title">
          <string>用户个人信息</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>31</y>
            <width>212</width>
            <height>141</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>幼圆</family>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>用户名：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="usernameLineEdit">
               <property name="styleSheet">
                <string notr="true">QLineEdit
{
    background:white;
    border-radius:3px;
    padding-left:3px;
    border: 1px solid rgb(209 , 209 , 209);
}</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <family>幼圆</family>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>密码：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="passwordLineEdit">
               <property name="styleSheet">
                <string notr="true">QLineEdit
{
    background:white;
    border-radius:px;
    padding-left:px;
    border: 1px solid rgb(209 , 209 , 209);
}</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <family>幼圆</family>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>性别：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="sexComboBox">
               <property name="font">
                <font>
                 <family>幼圆</family>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QComboBox
{
    background:white;
    border-radius:3px;
    padding-left:3px;
    border: 1px solid rgb(209 , 209 , 209);
}</string>
               </property>
               <property name="frame">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>男</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>女</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget1">
          <property name="geometry">
           <rect>
            <x>293</x>
            <y>31</y>
            <width>199</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <family>幼圆</family>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>年龄：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ageLineEdit">
               <property name="styleSheet">
                <string notr="true">QLineEdit
{
    background:white;
    border-radius:3px;
    padding-left:3px;
    border: 1px solid rgb(209 , 209 , 209);
}</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="font">
                <font>
                 <family>幼圆</family>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>受教育程度：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="educationComboBox">
               <property name="font">
                <font>
                 <family>幼圆</family>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QComboBox
{
    background:white;
    border-radius:3px;
    padding-left:3px;
    border: 1px solid rgb(209 , 209 , 209);
}</string>
               </property>
               <property name="frame">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>请选择</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>博士后</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>博士</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>硕士</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>本科</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>大专</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>高中</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>中专</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>初中</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>小学</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>小学以下</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;幼圆&quot;;</string>
         </property>
         <property name="title">
          <string>用户学习记录</string>
         </property>
         <widget class="QTabWidget" name="tabWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>551</width>
            <height>311</height>
           </rect>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="educationsystem.qrc"/>
 </resources>
 <connections/>
</ui>
